morphir-ir/shared/src/main/scala/zio/morphir/ir/PackageModule.scala

```diff
@@ -19,13 +22,48 @@ object PackageModule {
       }
       Specification(modules)
     }
+
+    def lookupModuleDefinition(path: Path): List[Option[ModuleDef[Annotations]]] = {
```

Comment:
Since we use ModuleName instead of Path for this, the implementation and return type is different than the original Elm. I decided to make this a List instead, let me know if there is a different way we want this to be expressed.

Reply:
You mind adding what the Elm signature is here?